.gitlab/macrobenchmarks.yml

```diff
@@ -152,7 +152,7 @@ appsec-disabled-iast-worst-ep-disabled:
   variables:
     DD_BENCHMARKS_CONFIGURATION: only-tracing
     BP_PYTHON_SCENARIO_DIR: flask-realworld
-    BP_ENDPOINT: http://0.0.0.0:8000/iast/propagation?string1=Hi&password=root1234
+    BP_ENDPOINT: http://0.0.0.0:8000/iast/articles?string1=Hi&password=root1234
```

Comment:
NB: For my lack of context... What is the difference with these endpoints?

Reply:
Sorry, I didn't explain. The second one includes all the logic in other scenarios (retrieving data from database, etc) while the first doesn't so we can't compare properly the overhead.

This would be the relevant PR to see: https://github.com/DataDog/flask-realworld-example-app/pull/19/files